### PR TITLE
972 derrive docs

### LINKED ISSUE #972: Documentation:  Clarify constant_parameter and constant_by_vars in derive_derived_param()
### Please select a category the issue is focused on?

Function Documentation

### Let us know where something needs a refresh or put your idea here!

Clarify `constant_parameter` and `constant_by_vars` in `derive_derived_param()`.
For example:
- move example from `constant_by_vars` to `constant_parameters`
- provide more details for `constant_parameters` either in the parameter description of in the details section
- reference the second example in the description of `constant_parameters` and `constant_by_vars`.

### PR DESCRIPTION
**Pull request by gg106046**
Close #972 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: R/derive_derived_param.R [hunk 1]
```diff
@@ -41,19 +41,18 @@
...
 #'
 #' @param constant_by_vars By variables for constant parameters
 #'
-#'   The constant parameters are merged to the other parameters using the
-#'   specified variables. This is useful if some parameters were measured only
-#'   once. For example if BMI should be derived and height is measured only once
-#'   while weight is measured at each visit. Height could be specified for the
-#'   constant parameters.
+#'   The constant parameters (parameters that are measured only once) are merged
+#'   to the other parameters using the specified variables. (Refer example 2)
```
Comment (bms63) [inline R/derive_derived_param.R:55]:
```suggestion
#'   to the other parameters using the specified variables. (Refer to Example 2)
```

File: R/derive_derived_param.R [hunk 2]
```diff
@@ -92,7 +91,7 @@
 #' @export
 #'
 #' @examples
-#' # derive MAP
+#' # Example 1: derive MAP
```
Comment (bms63) [inline R/derive_derived_param.R:94]:
```suggestion
#' # Example 1: Derive MAP
```

File: R/derive_derived_param.R [hunk 3]
```diff
@@ -117,7 +116,7 @@
 #'   )
 #' )
 #'
-#' # derive BMI where height is measured only once
+#' # Example 2: derive BMI where height is measured only once
```
Comment (bms63) [inline R/derive_derived_param.R:119]:
```suggestion
#' # Example 2: Derive BMI where height is measured only once
```

**Comment by gg106046** [comment]
> Hey @gg106046 Remember to put your issue number in the Committ message #972

Sure Ben! Will keep that in mind next time.

**Review by bms63 — APPROVED** [review]
Thanks Gayatri!

**Comment by bms63** [comment]
Can you see if you can link the issue on the right side in the Development box?

**Comment by gg106046** [comment]
Linked the issue #972!

**Comment by gg106046** [comment]
So the 'Man Pages/ Roxygen' check is still failing. Getting an error due to this issue --> "Can't fi
nd generic `sew` in package knitr to register S3 method." I did what Thomas Neitmann suggested on slack for this, but that still seems to fail. Am I missing something?

**Review by bms63 — APPROVED** [review]

**Comment by bms63** [comment]
Yay!  Your PR is done!

**Pull request merged**